### ISSUE TITLE
not receiving blur or focus events

### ISSUE THREAD

**Issue by jemmyw** [TylorS/cycle-snabbdom#21]
blur and focus events don't fire. In this simple example the focus event will never cause the dom to update, however the click event will:

``` javascript
import {Observable} from 'rx';
import {div, a, form, textarea} from 'cycle-snabbdom';

const main = sources => {
  const focus$ = Observable.merge(
    sources.DOM.select('.text').events('focus').map(true),
    sources.DOM.select('.text').events('focus').map(false)
  ).startWith(false);

  const click$ = sources.DOM.select('.text').events('click').map(true).startWith(false);

  const vtree$ = Observable.combineLatest(focus$, click$, (focus, clicked) =>
    div('.container', [
      div('.focus', [focus ? 'yes' : 'no']),
      div('.clicked', [clicked ? 'yes' : 'no']),
      form([
        textarea('.text')
      ])
    ])
  );

  return {
    DOM: vtree$
  }
};

export default main
```

**Comment by TylorS** [comment]
I think I have a small bug, because this _should_ work automatically. As a temporary workaround you can use

``` js
DOM.select().events('focus', {useCapture: true})
```

**Comment by TylorS** [comment]
Fix should be in v1.2.0
